rc/extra/clang.kak

```diff
@@ -136,11 +136,11 @@ define-command -hidden clang-show-error-info %{
         shift # skip timestamp
         for error in "$@"; do
```

Comment:
How about

```
desc=$(for error in "$@"; do ...)
```

I belive this would only eat the last newline, which we dont really care about.